src/components/ItemFilters/ItemFilters.tsx

```diff
@@ -47,11 +47,15 @@ const ItemFilters = ({
   const [invalidYear, setInvalidYear] = useState(false)
 
   const filterData = useRef<ItemFilterData[]>(
-    itemAggregations.map((aggregation: Aggregation) => {
-      if (aggregation.field === "location")
-        return new LocationFilterData(aggregation)
-      else return new ItemFilterData(aggregation)
-    })
+    itemAggregations
+      .filter((aggregation: Aggregation) => aggregation.field !== "format")
```

Comment:
@nonword should we remove format from item aggregations if we are not using them currently on the front end?